### PR TITLE
Replay

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

**Review by boks1971 — COMMENTED** [review]
Is this okay in oss? Guess it is needed here?

File: protobufs/livekit_internal.proto [hunk 1]
```diff
@@ -142,8 +142,9 @@ message RoomInternal {
   AutoTrackEgress track_egress = 1;
   AutoParticipantEgress participant_egress = 2;
   PlayoutDelay playout_delay = 3;
-  repeated CreateAgentJobDefinitionRequest agents = 5; 
+  repeated CreateAgentJobDefinitionRequest agents = 5;
```
Comment (biglittlebigben) [inline protobufs/livekit_internal.proto:145]: This type name changed in main.

**Review by biglittlebigben — APPROVED** [review]

**Review by boks1971 — APPROVED** [review]

File: protobufs/cloud_replay.proto [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  string src_room_id = 1;
+  string dest_room_name = 2;
+  int64 starting_pts = 3;
+}
+
+message LoadReplayResponse {
+  string replay_id = 1;
+}
+
+message RoomSeekRequest {
```
Comment (paulwe) [inline protobufs/cloud_replay.proto:27]: should this have replay somewhere in the name?

**Review by paulwe — APPROVED** [review]

**Review by davidzhao — APPROVED** [review]
lg!

**Pull request merged**